paper/src/main/java/vg/civcraft/mc/citadel/command/Activity.java

```diff
@@ -0,0 +1,48 @@
+package vg.civcraft.mc.citadel.command;
+
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import vg.civcraft.mc.citadel.Citadel;
+import vg.civcraft.mc.citadel.activity.ActivityMapStat;
+
+@CommandAlias("am")
+public class Activity extends BaseCommand {
+	@CommandAlias("stat")
+	@Description("Shows ActivityMap statistics.")
+	@CommandPermission("citadel.admin")
+	public void execute(CommandSender sender) {
+		ActivityMapStat stat = Citadel.getInstance().getActivityMap().getStat();
+		if (stat == null) {
+			sender.sendMessage(ChatColor.RED + "ActivityMap is disabled");
+			return;
+		}
+
+		sender.sendMessage(ChatColor.YELLOW + "[ActivityMap] object statistics:");
```

Comment:
As below, recommend usage of kyoris adventure over ChatColor

Reply:
Done: https://github.com/CivMC/Citadel/pull/39/commits/b83610d297d9730adf41d1a3bafb6d8ddfdd4de1#diff-2cdb8096b105724589cdb24355cc5f21234459e9ea138d0ff8c530d161b7b575R26